lib/model.js

```diff
@@ -1857,21 +1857,13 @@ class Model {
       countOptions.attributes = undefined;
     }
 
-    const countQuery = this.count(countOptions);
-    const findQuery = this.findAll(options);
-
```

Comment:
Both promise start executing as soon as possible, but we still want to keep `count === 0` logic

Reply:
You mean:
```
if (count === 0) {
	return {count, rows: []};
} else {
	return {count, rows};
}
```
? Or do you want to see findQuery.then inside countQuery.then?
I do not understand the meaning